solidity/ecdsa/tasks/initialize.ts

```diff
@@ -0,0 +1,57 @@
+import { task, types } from "hardhat/config"
+import {
+  TASK_INITIALIZE,
+  TASK_AUTHORIZE,
+  TASK_REGISTER,
+} from "@keep-network/random-beacon/export/tasks/initialize"
+import {
+  authorize,
+  register,
+} from "@keep-network/random-beacon/export/tasks/utils"
+
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+
+const TASK_AUTHORIZE_ECDSA = `${TASK_AUTHORIZE}:ecdsa`
+const TASK_REGISTER_ECDSA = `${TASK_REGISTER}:ecdsa`
+
+task(TASK_INITIALIZE, "Initializes staking for an operator").setAction(
+  async (args, hre, runSuper) => {
+    // Run initialization task from @keep-network/random-beacon.
+    await runSuper(args)
```

Comment:
If we were about to call task `initialize` from the script for beacon and ecdsa, then `mint` and `approve` will be called twice. I think we need to call `mint` and `approve` once, but `authorize` and `register` per application. We can call `initialize` from the beacon and call `TASK_AUTHORIZE_ECDSA` and `TASK_REGISTER_ECDSA` separately, but I take that the idea behind this PR was grouping the tasks into a single `initialize` task.

Reply:
The `mint` and `approve` tasks are implemented in a such manner that if `initialize` is executed for the second time it will not perform any actions if the current state is fine.

So if we call `initialize` in the random-beacon and then in ecdsa minting and approval will execute transactions just once.

On the other hand if we run `initialize` in the ecdsa only, then it will execute `mint` and `approval` to get to the desired state. In such case, there is no need to call `initialize` in the beacon.

Reply:
> On the other hand if we run initialize in the ecdsa only, then it will execute mint and approval to get to the desired state. In such case, there is no need to call initialize in the beacon.

Right, and this part doesn't feel right to me. IMO the `initialize` task in `ecdsa` project should handle the initialize tasks in `ecdsa` and probably not trigger the `initialize` task in `beacon`. I see that we have a flag for this `cd ecdsa/solidity && npx hardhat initialize --skip-beacon` but not sure if it should be there in the first place when calling it from the outside. Ecdsa depends on the beacon, but these are 2 applications where some parts are independent.

Anyway, I guess we can leave it for now and I would add a TODO to fix it later, but if you feel it's okay as-is, then up to you.